docs/schema.md

```diff
@@ -830,8 +830,8 @@ Only **a single type** (either `introduced`, `fixed`, `last_affected`,
 `limit`) is allowed in each event object. For instance,
 `{"introduced": "1.0.0", "fixed": "1.0.2"}` is **invalid**.
 
-Entries in the `events` array can contain either `last_affected` or `fixed`
-events, but not both. It's **strongly recommended** to use `fixed` instead of
+An events array can have entries containing either "last_affected" or "fixed" events, 
```

Comment:
Yes, there's some semantic nuance here, which is a little tricky to convey, I think.

So to get all JSON-technical, the `events` array is an array of JSON (event) _objects_, and the intention of the statement under scrutiny is to refer to these event objects, as you've surmised.

https://github.com/ossf/osv-schema/blob/main/validation/schema.json#L92-L150

```suggestion
Entries in the `events` array may be "last_affected" or "fixed" events, 
```

Reply:
@andrewpollock I think that it's possible that someone may still misread that suggestion as disallowing this case:

```
[..., {"last_affected": "1.0.0", "fixed": "1.0.1"}, ...]
```